bin/cylc-remove

```diff
@@ -58,7 +58,8 @@ def main():
         options.port, options.db, my_uuid=options.set_uuid,
         print_uuid=options.print_uuid)
     items, compat = parser.parse_multitask_compat(options, args)
-    pclient.put_command('remove_task', items, compat, options.spawn)
+    pclient.put_command('remove_task', items, compat,
+                        None, not options.no_spawn)
```

Comment:
Note the new arg `None` here.  It corresponds to "_=None" in the server-side function, which I think is a back-compat placeholder for the family indicator flag in the older command line interface.